tests/lowering/eltwise/binary/test_div.py

```diff
@@ -35,6 +35,10 @@ def test_div(device, input_shapes):
     assert target.count(ttnn.mul) == 1
     assert target.index(ttnn.reciprocal) < target.index(ttnn.mul)
     assert nodes[target.index(ttnn.mul)].args[1].target == ttnn.reciprocal
+    # Intermediate node meta check if preserved
+    for node in nodes:
```

Comment:
Any idea how this can be added centrally so one don't have to check it like this in every test?